### PR TITLE
Paging-3 implementation in SearchArticles Screen

### PR DESCRIPTION
**Pull request by avidraghavMM**

### CODE REVIEW

File: app/src/main/java/com/raghav/spacedawn/ui/fragments/SearchArticleFragment.kt [hunk 1]
```diff
@@ -40,7 +32,7 @@ class SearchArticleFragment : Fragment(R.layout.fragment_search_article) {
         binding = FragmentSearchArticleBinding.bind(view)
         setupRecyclerView()
 
-        searchArticlesAdapter.setOnItemClickListener {
+        articlesAdapter.setOnItemClickListener {
             val bundle = Bundle().apply {
                 putSerializable("article", it)
```
Comment (aditya-bhawsar-mm) [inline app/src/main/java/com/raghav/spacedawn/ui/fragments/SearchArticleFragment.kt]: It is advised to use the string literal here from a const in utils

File: app/src/main/java/com/raghav/spacedawn/utils/AlarmBroadCastReciever.kt [hunk 2]
```diff
@@ -34,27 +33,25 @@ class AlarmBroadCastReciever : BroadcastReceiver() {
         }
 
         val notification = NotificationCompat.Builder(p0!!, CHANNEL_ID)
-            .setContentTitle("Reminder!")
-            .setContentText("Reminder to see the Rocket Launch set via Space Dawn")
+            .setContentTitle(Constants.REMINDER)
```
Comment (aditya-bhawsar-mm) [inline app/src/main/java/com/raghav/spacedawn/utils/AlarmBroadCastReciever.kt]: bring this with R.strings please

**Review by aditya-bhawsar-mm — APPROVED** [review]
Approved but please address the comments and then merge

File: app/src/main/java/com/raghav/spacedawn/ui/fragments/SearchArticleFragment.kt (hunk 1 above)
Comment (avidraghavMM) [reply app/src/main/java/com/raghav/spacedawn/ui/fragments/SearchArticleFragment.kt]: done

File: app/src/main/java/com/raghav/spacedawn/utils/AlarmBroadCastReciever.kt (hunk 2 above)
Comment (avidraghavMM) [reply app/src/main/java/com/raghav/spacedawn/utils/AlarmBroadCastReciever.kt]: I pushed the code after making the prescribed change but will have to see why it's still showing the old code

Comment (avidraghavMM) [reply app/src/main/java/com/raghav/spacedawn/utils/AlarmBroadCastReciever.kt]: resolved

**Pull request merged**